pyracing/client.py

```diff
@@ -782,3 +784,11 @@ async def yearly_stats(self, cust_id):
             return []
 
         return [career_stats.YearlyStats(x) for x in response.json()]
+    
+    @staticmethod
+    def cleanup_payload(payload):
```

Comment:
This should probably go in helpers, but have we confirmed that this will still work for all of the endpoints? I know we have seen a few where leaving out the key works as well as sending null, but iracing is notoriously inconsistent and I am not super comfortable merging this until we can confirm that 100% of the endpoints operate this way. @Esterni

Reply:
Agreed. However, just stepping through what it's doing:
Every endpoint method sends a payload.
Every payload is a dictionary Key:Value pair.
A value of None is treated equally to iRacing as if it wasn't there at all. 

The ones with 'null' are explicitly set as iRacing won't return results otherwise, but I haven't tested each endpoint by hand and removed parameters 1 by 1